src/main/java/mate/academy/bookshop/dto/user/UserRegistrationRequestDto.java

```diff
@@ -9,7 +9,7 @@
 @Data
 @FieldMatch
 public class UserRegistrationRequestDto {
-    private static final String NOT_BE_NULL = "Don't be null";
+    private static final String NOT_BE_NULL = "Can't be null";
```

Comment:
For the future, it is not recommended to post changes to the PR that do not relate to the task. I suggest making a separate request with such changes.

Reply:
++